packages/ember-metal/lib/map.js

```diff
@@ -33,10 +34,10 @@ function missingNew(name) {
 }
 
 function copyNull(obj) {
-  var output = Object.create(null);
+  var output = new EmptyObject();
```

Comment:
nice, this should actually help improve some ED stuff, which relies on large numbers of Maps (and last i checked lots of copies/clones of them)